src/AutorestSwift/Models/Complex/ConstantSchema.swift

```diff
@@ -112,4 +112,36 @@ class ConstantSchema: Schema {
 
         try super.encode(to: encoder)
     }
+
+    /**
+     Convert the Constant into String format in Swift
+     */
+    public func formatValue(isSkipUrlEncoding: Bool) -> String {
```

Comment:
For documenting public functions, you should right-click, "Show Code Actions" > "Add Documentation". This will pre-populate the docstring with the fields in the right format.

Reply:
I would rename `isSkipUrlEncoding` to just `skipUrlEncoding`.